src/get_imports.rs

```diff
@@ -71,25 +87,26 @@ fn get_string_of_span<'a>(file_text: &'a Vec<u8>, span: &'a swc_common::Span) ->
 fn get_imports_map(
     imports: &Vec<SourceSpecifiers>,
     importer_file_path: &PathBuf,
-) -> HashMap<String, Option<HashSet<String>>> {
+) -> Result<HashMap<String, Option<HashSet<String>>>, GetImportError> {
     let mut imports_map: HashMap<String, Option<HashSet<String>>> = HashMap::new();
 
     imports.iter().for_each(|import| {
         let set: HashSet<String> = import
             .specifiers
             .iter()
+            .filter(|spec| spec.is_default() || spec.is_named())
             .filter_map(|spec| -> Option<String> {
-                let file_text = std::fs::read(importer_file_path).expect(&format!(
-                    "error opening source file \"{:?}\"",
-                    importer_file_path
-                ));
+                let file_text = match std::fs::read(importer_file_path) {
+                    Ok(text) => text,
+                    _ => {
+                        return None;
```

Comment:
If you follow my above comment about pre-parsing these in get_imports_from_file, you won't have to do this extra read or this weird _ => None.